### DISCUSSION TITLE
Rowy Calendar

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#1101] [Idea / Feature Request]
Having a Google calendar by the Rowy team and sharing the link with the contributors and users along with the discord events.
The calendar will allow having a single point of truth for upcoming office hours, discussions,s and more.
It would reduce the effort required by the contributor to manually add a calendar link to his/her calendar from discord events.

**Comment by BeeBombshell** [comment]
Hey @RajGM! Appreciate your thought regarding this idea. However, we already have a dedicated page for the upcoming Rowy events here: https://rowy.io/events. This page includes all the upcoming event details and agenda. You can add the event to your calendar, also sign up for email and SMS reminders.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/53828745/215428057-49adba7c-0ac8-43e5-a30f-68b932b3564d.png">

**Comment by RajGM** [comment]
Hi @BeeBombshell 
It's great that we have a Luma page but it feels annoying to visit the page and manually opt for events.
That's why I think Google Calendar can supplement the Luma one.
